src/k8s-engine/k3sHelper.ts

```diff
@@ -35,6 +38,98 @@ import { showMessageBox } from '@/window';
 
 const console = Logging.k8s;
 
+// kubernetes-client/javascript doesn't have support for the `proxy-url` cluster field.
```

Comment:
Consider moving overrides for `kubernetes-client/javascript` to a separate file to reduce the amount of noise in `k3sHelper`.